enterprise/static/enterprise/grant_data_sharing_permissions.js

```diff
@@ -1,17 +1,43 @@
+function togglePolicy (force_open) {
+    var policy = document.getElementsByClassName("consent-policy")[0];
+    currentDisplay = policy.style.display;
+    var icon = document.getElementById("consent-policy-dropdown-icon");
+    if ((currentDisplay === "none") || force_open) {
+        policy.style.display = "block";
+        icon.className = "fa fa-chevron-down fa-stack-1x";
+        location.href = "#";
+        location.href = "#consent-policy-dropdown-bar";
```

Comment:
Why is href set to `#` right before being set to `#consent-policy-dropdown-bar`? Should the `location.href = "#";` line be moved into the `else` clause below?